tests/support/expectations.js

```diff
@@ -661,16 +664,6 @@ module.exports = function (request, config) {
       }
     },
     'Header setting': {
-      'setting mocked headers': function (test) {
```

Comment:
Because I removed the set hook. This test became redundant, testing only the hook it created.